tests/utils/utils.go

```diff
@@ -204,6 +204,7 @@ func CreateDefaultRelayerConfig(
 		MetricsPort:            9090,
 		SourceBlockchains:      sources,
 		DestinationBlockchains: destinations,
+		APIPort:                1234,
```

Comment:
Any reason not to use the default port 8080? That's what will be used if this field is not explicitly set.

Reply:
I just needed to use something. The defaults don't get set in the tests. I've changed it to 8080